node_modules/oae-content/lib/search.js

```diff
@@ -166,76 +166,77 @@ ContentAPI.on(ContentConstants.events.DELETED_COMMENT, function(ctx, comment, co
 /**
  * Produce the necessary content comment search documents.
  *
- * @param  {Object[]}   resources           The resources representing the content item whose comments to index
- * @param  {Function}   callback            Standard callback function
- * @param  {Object}     callback.err        An error that occurred, if any
- * @param  {Object[]}   callback.documents  The content comment search documents
+ * @see SearchAPI#registerChildSearchDocument
  * @api private
  */
-var _produceContentCommentDocuments = function(resources, callback, _documents) {
+var _produceContentCommentDocuments = function(resources, callback, _documents, _errors) {
     _documents = _documents || [];
+    _errors = _errors || [];
```

Comment:
We should try and keep this as being falsey until an error actually happens. Doing `_.union(_errs, errs)` should still work if `_errs` is undefined so that shouldn't have to change (can you quickly verify that in the REPL? I'm pretty sure I did that for group delete stuff)